tip-wlan/charts/wlan-prov-service/templates/deployment.yaml

```diff
@@ -169,6 +169,13 @@ spec:
             failureThreshold: {{ .Values.probes.readinessProbe.failureThreshold }}
             periodSeconds: {{ .Values.probes.readinessProbe.periodSeconds }}
             successThreshold: {{ .Values.probes.readinessProbe.successThreshold }}
+          startupProbe:
+            httpGet:
+              path: /ping
+              port: {{ .Values.service.port2 }}
+              scheme: {{ .Values.probes.readinessProbe.scheme }}
+            failureThreshold: 500
```

Comment:
The probe parameters shouldn't be hard coded into the template. We can move them to the values of that chart with reasonable defaults.

Reply:
there is a very little upside to this, as this is just a big timeout before we want it to fail

Reply:
And yet, I think this should be parametrized and exposed through values for future fine tuning if someone needs it.